README.md

```diff
@@ -90,19 +90,25 @@ corrections in tag values or units.
 Here is a summary of the types of changes that correspond to different
 levels of changes in the semantic version:
 
-| Change                          | Semantic-level | 
-| ---------------------------------- | -------------- |
-| Major addition to HED functionality     | Major  |
-| Tag deleted from schema.                | Major  |
-| Unit or unit class removed from node.   | Major  |
-| New tag added to the schema.            | Minor  |
-| New attribute added to schema.          | Minor  |
-| New unit class or unit added to schema. | Minor  |
-| New unit class added to node.           | Minor  |
-| Node moved in schema without change in meaning. | Minor |
-| Revision of description field in schema.        | Patch   |
-| Correction of suggestedTag or relatedTag.       | Patch  |
-| Correction of wiki syntax such as closing tags. | Patch |
+| Change                                              | Semantic-level |
+|-----------------------------------------------------|----------------|
+| Major addition to HED functionality                 | Major          |
+| Tag deleted from schema.                            | Major          |
+| Unit or unit class removed from node.               | Major          |
+| Node attribute value changed (excluding below)      | Major          |
```

Comment:
I don't think this should trigger a major version change

Reply:
Node attribute values could include things like removing a valueClass, making a unit a prefix now, or other quirky behaviors.  

I just didn't think it was worth trying to explicitly cover all cases, given most are very unlikely.(e.g. removing a modifier could also have catastrophic effects without notification, but is all but assured to never happen)

I'm not very opinionated, just that was the thought behind why I marked in major.

Reply:
I'm going to merge and we can review again.